modules/ROOT/pages/managing-apis.adoc

```diff
@@ -55,7 +55,7 @@ To remove APIs from your portal:
 +
 In the *APIs in the portal* tab, the API is removed from the list.
 
-== Manage API Versions Visible
+== Make API Versions Visible
```

Comment:
this heading change is because line 38, https://github.com/mulesoft/docs-api-experience-hub/blob/latest/modules/ROOT/pages/managing-apis.adoc?plain=1#L38C216-L38C271 references it in that name